scrowl-io/owl-ui
# Feature/grid system

### Short summary

Added a basic grid system using flexbox, the design was inspired from http://flexboxgrid.com/

---

### Test steps

1. `yarn start:web`
2. go to `http://localhost:1234`
3. you should see a row with 3 columns

The page source is `web/src/app.tsx`.
